src/plugins/workspace/public/types.ts

```diff
@@ -39,7 +39,7 @@ export interface DataSourceAttributesWithWorkspaces extends Omit<DataSourceAttri
   workspaces?: string[];
 }
 
-export type WorkspaceCollaboratorPermissionType = 'user' | 'group';
+export type WorkspaceCollaboratorPermissionType = 'user' | 'group' | 'unknown';
```

Comment:
I think this unknown type only been used in test cases. I'm prefer not update the type here. Add `as unknown as xx` would be fine in the test case.